### PR TITLE
How does the name "sbt" stand for and shouldn't it be written "SBT"?

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: src/reference/05-Faq/00.md [hunk 1]
```diff
@@ -30,18 +30,25 @@ Frequently Asked Questions
...
-it was called *Simple Build Tool*, but when the first release came out,
-it was already called [sbt 0.3.2](http://www.scala-lang.org/old/node/392.html).
+**TL;DR** the name sbt doesn't stand for anything, it's just "sbt", and it should be written that way.
 
-sbt is a proper noun that must be spelled with all lowercase.
-Any deviation from the guideline such as title casing or all caps are subject to side-eye frown face.
-We occationally set `sbt` in monospace font to specifically mean the bash/batch script `sbt` command.
+When Mark Harrah ([@harrah][]) first created the project he called it "Simple Build Tool", but in his
+[first public announcement][sbt 0.3.2 announcement] of it he already referred to it as just "sbt".
+Over time sbt was sometimes defined as "Scala Build Tool", but we believe that isn't accurate either
```
Comment (eed3si9n) [inline src/reference/05-Faq/00.md]: "defined"? I don't think neither Mark nor Lightbend has ever called it Scala Build Tool.

**Review by eed3si9n — CHANGES_REQUESTED** [review]
Overall nice rewrite.

File: src/reference/05-Faq/00.md (hunk 1 above)
Comment (dwijnand) [reply src/reference/05-Faq/00.md]: Right. I mean "some people". I'll tweak.

File: src/reference/05-Faq/00.md [hunk 2]
```diff
@@ -30,18 +30,25 @@ Frequently Asked Questions
...
-#### What does sbt stand for, and how should I spell it?
+#### What does the name "sbt" stand for and shouldn't it be written "SBT"?
 
-sbt is just sbt: It's no longer an acronym. When the project was first started
-it was called *Simple Build Tool*, but when the first release came out,
-it was already called [sbt 0.3.2](http://www.scala-lang.org/old/node/392.html).
+**TL;DR** the name sbt doesn't stand for anything, it's just "sbt", and it should be written that way.
 
-sbt is a proper noun that must be spelled with all lowercase.
-Any deviation from the guideline such as title casing or all caps are subject to side-eye frown face.
```
Comment (eed3si9n) [inline src/reference/05-Faq/00.md:40]: Why was this taken out?

Comment (eed3si9n) [reply src/reference/05-Faq/00.md:40]: Title casing is something I have to argue against even in-house.

Comment (dwijnand) [reply src/reference/05-Faq/00.md:40]: Didn't like it.

**Pull request merged**